git-publish

```diff
@@ -553,9 +553,15 @@ def main():
         tag(tag_name_staging(topic), tag_message, annotate=anno, force=True)
 
     prefix = options.prefix
-    if prefix is None:
+    if prefix is not None:
+        # when prefix == "", it will clear per-branch prefix
```

Comment:
I understand it clears the prefix but it's hacky.  It doesn't actually remove the prefix from the .gitconfig.  Instead it stores an empty string in .gitconfig and relies on the change from "if prefix is not None" to "if not prefix" so git-publish ignores the empty string.

Adding --clear-subject-prefix is cleaner.  Or just document that it can be cleared with "git config --unset branch.mybranch.gitpublishsubjectprefix".

The per-branch subject prefix behavior needs to be documented in README.rst anyway so I would just show the git config --unset command.